logs-lib/README.md

```diff
@@ -0,0 +1,140 @@
+# Logs lib
+
+This logs lib can be used to generate logs dashboard using NEW [grafonnet](https://github.com/grafana/grafonnet).
+
+## Import
+
+```sh
+jb init
+jb install https://github.com/grafana/jsonnet-libs/logs-lib
+jb install https://github.com/grafana/grafonnet/gen/grafonnet-latest@main
+```
+
+## Examples
+
+### Generate kubernetes logs dashboard
+
+```jsonnet
+local logs = import 'github.com/grafana/jsonnet-libs/logs-lib/logs/main.libsonnet';
+{
+
+  _config+:: {
```

Comment:
personally, I would drop the underscore. It is associated for me with a set of not-very-good Jsonnet practices.

Reply:
just following mixin format we still use as of today.

Reply:
Yah, I would encourage you to break free from that :-)

Reply:
Agreeing with Malcolm here, rather than incorporating `_config` into this to fit in with the old-style I'd suggest documenting the functions like you do below may be sufficient for people that want to use your library. Also considering that this is not a mixin on itself but rather something that can be used to create a mixin, this means we don't need to adhere to the old-style for anyone.